### PR TITLE
Feature/sexy ass branch

### PR DESCRIPTION
**Pull request by katlazarova**

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -98,6 +98,41 @@ body {
   margin: 0 auto;
 }
 
+.card-component-container {
+  max-width: 80%;
```
Comment (ElXyfer) [inline css/style.css]: You don't need max-width here.

File: css/style.css [hunk 2]
```diff
@@ -98,6 +98,41 @@ body {
...
+  display: block;
+}
+
+.card {
+  display: flex; 
+  flex-direction: column;
+  position: relative;
+  box-sizing: border-box;
+  text-anchor: middle;   
+}
```
Comment (ElXyfer) [inline css/style.css]: You don't need box-sizing and text-anchor.

File: css/style.css [hunk 3]
```diff
@@ -98,6 +98,47 @@ body {
...
+
+.illustrative-image {
+  object-fit: cover;
+  width: 300px;
+  height: 200px;
+}
```
Comment (ElXyfer) [inline css/style.css]: Change width and height on the above to be 100%.

File: index.html [hunk 4]
```diff
@@ -36,6 +36,40 @@ <h2>Donating To Us</h2>
 		<div class="content-divider">
 		</div> <!-- End of content-divider -->
 
+		<div class="card-component-container">
+			<h2>More Ways To Help</h2>
```
Comment (ElXyfer) [inline index.html]: This will look better centred (through css).

File: index.html [hunk 5]
```diff
@@ -36,6 +36,40 @@ <h2>Donating To Us</h2>
...
+			<div class="row-container">
+				<div class="card-wrapper">
+					<div class="card card-one">
+						<a href="#"><img src="img/volunteer.jpeg" alt="volunteers" class="illustrative-image"></a>
```
Comment (ElXyfer) [inline index.html]: Give classes to anchor tags and style them. `width: 100%` and `height: 225px` will look good.

File: css/style.css [hunk 6]
```diff
@@ -98,6 +98,47 @@ body {
...
+
+.card {
+  display: flex; 
+  flex-direction: column;
+  position: relative;
+  box-sizing: border-box;
+  text-anchor: middle;   
+}
+
+.illustrative-image {
```
Comment (ElXyfer) [inline css/style.css]: Perhaps update this to "card-image" to be constant?

File: index.html (hunk 5 above)
Comment (ElXyfer) [reply index.html]: Perhaps update illustrative-image class to "card-image" to be constant? Class in css would also need updating.

File: css/style.css (hunk 1 above)
Comment (katlazarova) [reply css/style.css]: I don't think so.... you'll see when I finish styling, but I'll consider it

File: css/style.css (hunk 3 above)
Comment (katlazarova) [reply css/style.css]: when trying this, the images were too big... doesn't work with object-fit: cover

Comment (ElXyfer) [reply css/style.css]: This because you need to add `height: 225px;` and `width: 100%;` on the image container too.

File: index.html (hunk 5 above)
Comment (katlazarova) [reply index.html]: done

File: index.html (hunk 4 above)
Comment (katlazarova) [reply index.html]: agreed and done

File: css/style.css (hunk 3 above)
Comment (katlazarova) [reply css/style.css]: done

File: css/style.css (hunk 2 above)
Comment (katlazarova) [reply css/style.css]: done

File: css/style.css [hunk 7]
```diff
@@ -99,11 +101,17 @@ body {
 }
 
 .card-component-container {
-  max-width: 80%;
+  width: 90%;
```
Comment (ElXyfer) [inline css/style.css]: The value of this could be `var(--gutter-size)`

**Pull request merged**